src/lpwFuncs.cpp

```diff
@@ -232,37 +234,12 @@ void dormantUntilInterrupt(int interruptPin) {
  * @brief Handles inactivity by transitioning the system to dormant mode.
  */
 void inactivityHandler() {
-  inactivityTime=0;
-  deinitBattery();
-  #ifndef FLIPMOUSE
-    MouseBLE.end();     // turn off Bluetooth
-  #endif
-  #ifdef DEBUG_BATTERY_MANAGEMENT
-    Serial.println("goodbye, going to sleep!");
-  #endif 
-  Serial.flush();
-  Serial.end();
-  displayMessage((char*)"ByeBye");
-  delay(2000);   // time for the user to read the message
-  disable3V3();  // shut down peripherals
```

Comment:
isn't this essential ??  
it shuts down the power supply external loads ...
where is this in your code - or why was it removed?